eng/tox/mypy_hard_failure_packages.py

```diff
@@ -14,4 +14,5 @@
   "azure-ai-formrecognizer",
   "azure-ai-metricsadvisor",
   "azure-eventgrid",
+  "azure-mixedreality-remoterendering",
```

Comment:
how advisable is it for us to be mandatory hard failure here? i found it useful to be able to run tox mypy

Reply:
This opts you into failing the CI build if there are mypy errors in your library. It is advisable to have a mypy "clean" library before release and is part of our pre-release checklist. If this isn't feasible with your current timeline you could just open an issue.

Reply:
okay - i have currently 0 mypy issues on the current state of the SDK and will continue to run it and make sure it stays like this. will stay opted in.